utils/terrascript_client.py

```diff
@@ -41,7 +43,11 @@
                                aws_cloudwatch_log_group, aws_kms_key,
                                aws_kms_alias,
                                aws_elasticsearch_domain,
-                               aws_iam_service_linked_role)
+                               aws_iam_service_linked_role,
+                               aws_lambda_function, aws_lambda_permission,
+                               aws_cloudwatch_log_subscription_filter)
+
+LAMBDA_GIT = 'https://raw.githubusercontent.com/app-sre/aws-lambda/master/'
```

Comment:
I would recommend renaming `aws-lambda` repository. The name is too generic. Maybe something like `cloudwatch-stream-elasticsearch-lambda`